### PR TITLE
PMM-13485 Fix ovf pipeline for v2

### PR DESCRIPTION
**Pull request by talhabinrizwan**
PMM-13485

Link to the Feature Build: SUBMODULES-3758

### CODE REVIEW

File: build/Makefile [hunk 1]
```diff
@@ -21,12 +21,13 @@ fetch-el9:
...
 		-o ${PACKER_CACHE_DIR}/id_rsa_vagrant
 	chmod 600 ${PACKER_CACHE_DIR}/id_rsa_vagrant
-	test -f ${PACKER_CACHE_DIR}/box/oracle9.ova \
-		|| curl -fL https://pmm-build-cache.s3.us-east-2.amazonaws.com/VBOXES/oracle9-202401.31.0.box -o ${PACKER_CACHE_DIR}/box/oracle9.ova
 
-	# NOTE: image from vagrant registry is twice as large
+	# Add the box using Vagrant
 	test -f ${PACKER_CACHE_DIR}/box/box.ovf \
-		|| tar -C ${PACKER_CACHE_DIR}/box -xvf ${PACKER_CACHE_DIR}/box/oracle9.ova
+		|| VAGRANT_HOME=${PACKER_CACHE_DIR}/box vagrant box add bento/oraclelinux-9 --box-version 202407.23.0 --provider virtualbox
```
Comment (ademidoff) [inline build/Makefile:27]:
```suggestion
		|| VAGRANT_HOME=${PACKER_CACHE_DIR}/box vagrant box add bento/oraclelinux-9 --box-version ${BOX_VERSION} --provider virtualbox
```

File: build/Makefile [hunk 2]
```diff
@@ -21,12 +21,13 @@ fetch-el9:
...
+
+	test -f ${PACKER_CACHE_DIR}/box/box.ovf \
+		|| cp -rp ${PACKER_CACHE_DIR}/box/boxes/bento-VAGRANTSLASH-oraclelinux-9/202407.23.0/amd64/virtualbox/* ${PACKER_CACHE_DIR}/box
```
Comment (ademidoff) [inline build/Makefile:30]:
```suggestion
		|| cp -rp ${PACKER_CACHE_DIR}/box/boxes/bento-VAGRANTSLASH-oraclelinux-9/${BOX_VERSION}/amd64/virtualbox/* ${PACKER_CACHE_DIR}/box
```

Comment (ademidoff) [reply build/Makefile:30]: Let's use vars for versions as we already do.

File: build/Makefile [hunk 3]
```diff
@@ -21,12 +21,13 @@ fetch-el9:
...
 		-o ${PACKER_CACHE_DIR}/id_rsa_vagrant
 	chmod 600 ${PACKER_CACHE_DIR}/id_rsa_vagrant
-	test -f ${PACKER_CACHE_DIR}/box/oracle9.ova \
-		|| curl -fL https://pmm-build-cache.s3.us-east-2.amazonaws.com/VBOXES/oracle9-202401.31.0.box -o ${PACKER_CACHE_DIR}/box/oracle9.ova
 
-	# NOTE: image from vagrant registry is twice as large
+	# Add the box using Vagrant
 	test -f ${PACKER_CACHE_DIR}/box/box.ovf \
-		|| tar -C ${PACKER_CACHE_DIR}/box -xvf ${PACKER_CACHE_DIR}/box/oracle9.ova
+		|| VAGRANT_HOME=${PACKER_CACHE_DIR}/box vagrant box add bento/oraclelinux-9 --box-version ${BOX_VERSION} --provider virtualbox
```
Comment (ademidoff) [inline build/Makefile:27]: @talhabinrizwan I think we need to define that variable on line 4 :)

**Review by BupycHuk — APPROVED** [review]

**Review by JiriCtvrtka — APPROVED** [review]

**Pull request merged**